dcs/bin/scripts/dcsbind.sh

```diff
@@ -354,7 +354,7 @@ dcsEcho "gv_float_external_ip :" $gv_float_external_ip
 dcsEcho "gv_float_internal_ip :" $gv_float_internal_ip
 
 #Check if AWS_CLOUD environment variable defined
-if [[ -z $AWS_CLOUD ]]; then
+if [[ $AWS_CLOUD == "false" ]]; then
```

Comment:
wasn't the earlier check ok ? Does AWS_CLOUD get set to false also or just true/empty ?